python/src/EfficientGlobalOptimization_doc.i.in

```diff
@@ -11,16 +11,15 @@ variance.
 Then the new point is evaluated using the original model and the metamodel is
 relearnt on the extended design of experiment.
 
-Available constructors:
-    EfficientGlobalOptimization(*problem, krigingResult*)
-
 Parameters
 ----------
 problem : :class:`~openturns.OptimizationProblem`
     The optimization problem to solve
-    optionally, a 2nd objective marginal can be used as noise
 krigingResult : :class:`~openturns.KrigingResult`
     The result of the meta-model on the first design of experiment
+noise : :class:`~openturns.Function`
+    Noise model used on design experiment points, and also for improvement optimization
+    unless a (possibly cheaper) model is set via :meth:`setNoiseModel`.
```

Comment:
```suggestion
```
This line is unnecessarily confusing. Perhaps something should be written in the "Notes" section about the distinction between "metamodel" noise and "criterion" noise.